openlibrary/plugins/recaptcha/recaptcha.py

```diff
@@ -51,7 +52,12 @@ def validate(self, value=None):
             response=i.recaptcha_response_field,
             remoteip=web.ctx.ip)
 
-        response = urllib.urlopen('http://www.google.com/recaptcha/api/verify', urllib.urlencode(data)).read()
+        # If HTTP call fails, let the user through
+        try:
+            response = urllib2.urlopen('http://www.google.com/recaptcha/api/verify', urllib.urlencode(data), timeout=2).read()
+        except urllib2.URLError:
```

Comment:
Do we also want to catch urllib2.HTTPError? i.e. if their server responds w/ 503 or other such responses?
Also, I wonder how this might be able to be abused... (e.g. forcing request to fail by sending too much data). Maybe the captcha doesn't have enough security value for such cases to matter?